src/main/java/org/mozilla/jss/nss/BufferProxy.java

```diff
@@ -11,10 +11,12 @@ public BufferProxy(byte[] pointer) {
      *
      * But this does it for you.
      */
+    @Override
     protected void releaseNativeResources() {
         Buffer.Free(this);
     }
 
+    @Override
```

Comment:
My understanding was that `finalize` wasn't strictly an override in all versions of JDK? I thought in the future (e.g., JDK17?) they were removing the finalize member on `Object`. Maybe I'm misremembering and they've not made plans to fully remove it? Might be worth looking into.

Reply:
Ah, wrong PR.